chronicles.nim

```diff
@@ -282,19 +296,12 @@ template log*(stream: type,
 template logFn(name, severity) {.dirty.} =
   template `name`*(eventName: static[string],
                    props: varargs[untyped]) {.dirty.} =
-
-    bind logIMPL, bindSym, brForceOpen
-    logIMPL(instantiationInfo(), activeChroniclesStream(),
```

Comment:
The reason why I have used these slightly repeated bodies is to shave few nanoseconds from the compile times. Haven't tried to measure the effect.

Reply:
> to shave few nanoseconds from the compile times

Readability before nano-optimisations ;-)